packages/core/test/tag/compoundTagTests.tsx

```diff
@@ -33,13 +34,24 @@ describe("<CompoundTag>", () => {
 
     it("renders icons", () => {
         const wrapper = shallow(
-            <CompoundTag icon="tick" rightIcon="airplane" leftContent="Hello">
+            <CompoundTag icon="tick" endIcon="airplane" leftContent="Hello">
                 World
             </CompoundTag>,
         );
         assert.lengthOf(wrapper.find(Icon), 2);
     });
 
+    it("prefers endIcon to rightIcon", () => {
+        const result = render(
```

Comment:
I typically opt to use [`container`](https://testing-library.com/docs/react-testing-library/api/#container) off of render to run query methods since it's executed closer to where the component is rendered. Could we use that instead?
```suggestion
        const { container } = render(
```
```ts
expect(container.querySelector(`.${Classes.ICON}-airplane`)).to.exist; 
expect(container.querySelector(`.${Classes.ICON}-add`)).not.to.exist;
```